admin/google_search_console/class-gsc-category-filters.php

```diff
@@ -0,0 +1,178 @@
+	 *
+	 * @var array
+	 */
+	private $filter_values   = array();
+
+	/**
+	 * The current view
+	 *
+	 * @var string
+	 */
+	private $current_view;
+
+	/**
+	 * Constructing this object
+	 *
+	 * Setting the hook to create the issues categories as the links
+	 *
+	 * @param string $screen_id
+	 * @param string $platform
+	 */
+	public function __construct( $screen_id, $platform ) {
+		add_filter( 'views_' . $screen_id, array( $this, 'as_array' ) );
+
+		$this->set_counts( $platform );
+	}
+
+	/**
+	 * Getting the current view
+	 */
+	public function current_view() {
+		$view        = ( $status = filter_input( INPUT_GET, 'category' )) ? $status : 'not_found';
+		$mapped_view = WPSEO_GSC_Mapper::category( $view );
+		if ( filter_input( INPUT_GET, 'category' ) === null && empty( $this->category_counts[ $mapped_view ] ) ) {
+			$view = WPSEO_GSC_Mapper::category( key( $this->category_counts ), true );
+		}
+
+		return $this->current_view = $view;
+	}
+
+	/**
+	 * Returns the current filters as an array
+	 *
+	 * Only return categories with more than 0 issues
+	 *
+	 * @return array
+	 */
+	public function as_array() {
+		$new_views = array();
+
+		foreach ( $this->category_counts as $category_name => $category ) {
+			$new_views[] = $this->create_view_link( WPSEO_GSC_Mapper::category( $category_name, true ), $category['count'] );
+		}
+
+		return $new_views;
+	}
+
+	/**
+	 * Setting the view counts based on the saved data. The info will be used to display the category filters
+	 *
+	 * @param string $platform
+	 */
+	private function set_counts( $platform ) {
+		$platform_counts = $this->get_counts();
+
+		if ( array_key_exists( $platform, $platform_counts ) ) {
+			$this->category_counts = $this->parse_counts( $platform_counts[ $platform ] );
+		}
+
+		$this->set_filter_values();
+	}
+
+	/**
+	 * Getting the options with the counts
+	 *
+	 * @return mixed
+	 */
+	private function get_counts() {
+		return get_option( WPSEO_GSC_Count::OPTION_CI_COUNTS, array() );
+	}
+
+	/**
+	 * Setting the values for the filter
+	 */
+	private function set_filter_values() {
+		$this->set_filter_value( 'access_denied', __( 'Access denied', 'wordpress-seo' ), __( 'Server requires authentication or is blocking Googlebot from accessing the site.', 'wordpress-seo' ) );
+		$this->set_filter_value( 'faulty_redirects', __( 'Faulty redirects', 'wordpress-seo' ) );
+		$this->set_filter_value( 'not_followed',__( 'Not followed', 'wordpress-seo' ) );
+		$this->set_filter_value( 'not_found', __( 'Not found', 'wordpress-seo' ), __( 'URL points to a non-existent page.', 'wordpress-seo' ) );
+		$this->set_filter_value( 'other', __( 'Other', 'wordpress-seo' ), __( 'Google was unable to crawl this URL due to an undetermined issue.', 'wordpress-seo' ) );
+		$this->set_filter_value( 'roboted', __( 'Blocked', 'wordpress-seo' ), __( 'Googlebot could access your site, but certain URLs are blocked for Googlebot-mobile for smartphones in your robots.txt file.', 'wordpress-seo' ) );
```

Comment:
=> Googlebot could access your site, but certain URLs are blocked for Googlebot in your robots.txt file. This block could either be for all Googlebots or even specifically for Googlebot-mobile.